### ISSUE TITLE
attach topics

### ISSUE THREAD

**Issue by johnabuck** [agilepro/cognoscenti#294]
We've gone back and forth on attaching topics to agenda items. For the facilitation class I did need to attached topics in a couple of items and had copy the URL and include in the Agenda Item Description. So, not urgent, but I would vote for being able to attach topics.

**Comment by agilepro** [comment]
Right, we have the single "linked topic" but that seems not to be working.  I know only one person who uses it and that is not me.

OK, let's move this to high priority.   It is not too hard.
